targets/ar71xx-generic/profiles.mk

```diff
@@ -1,5 +1,12 @@
 # List of hardware profiles
 
+## GL-iNet
+
+# GL-iNet 1.0 ; BB is the 8MB version, works on 16M too
```

Comment:
I'd like to backport the GL-iNET changes from CC (as it should be done anyways when we upgrade to CC) instead of using the 8M image for both versions.

Currently OpenWrt is missing code to disinguish the versions for the autoupdater, I'll whip up a patch to achieve that.